### DISCUSSION TITLE
Add plain text field UI element

### DISCUSSION THREAD

**Discussion by ryanleichty** [craftcms/cms#15664] [Ideas] [answered]
It would be great if you could add a plain text UI element that allows you to add generic instructions or text within a field layout. We often want to add accompanying text alongside a heading, so in order to get around that we create a custom template. It breaks the flow of building field layouts.

![CleanShot 2024-09-04 at 11 14 39@2x](https://github.com/user-attachments/assets/82c74f18-3923-4ebc-bc4f-96d45b7887cf)
![CleanShot 2024-09-04 at 11 12 28@2x](https://github.com/user-attachments/assets/2d4aad62-2122-4735-9464-b3b8dbf95de4)

**Comment by rynpsc** [comment]
I've done this in the past using a custom layout element (with markdown support vs plain text) but would be nice to see this in core.

**Comment by brandonkelly** [comment, marked as answer]
Added for Craft 5.5 🎉 (#15674)

**Reply by ryanleichty** [reply]
Amazing! +1 for the optional pane and +1 for markdown support 💯

**Reply by brandonkelly** [reply]
Craft 5.5.0 is out with the new Markdown UI element 🎉
